docs/en/Deployment.md

```diff
@@ -0,0 +1,159 @@
+ 
+### Then verify that the Name Server starts successfully
+$ tail -f ~/logs/rocketmqlogs/namesrv.log
+The Name Server boot success...
+```
+
+We can see 'The Name Server boot success.. ' in namesrv.log that indicates the NameServer has been started successfully.
+
+**2）Start Broker**
+
+```shell
+### Also start Broker first
+$ nohup sh bin/mqbroker -n localhost:9876 &
+
+### Then verify that the Broker is started successfully, for example, the IP of Broker is 192.168.1.2 and the name is broker-a
+$ tail -f ~/logs/rocketmqlogs/Broker.log 
+The broker[broker-a,192.169.1.2:10911] boot success...
+```
+
+We can see 'The broker[brokerName,ip:port] boot success..' in Broker.log that indicates the Broker has been started successfully.
+
+### 2 Multiple Master mode
+
+Multiple master mode means a mode with all master nodes(such as 2 or 3 master nodes) and no slave node. The advantages and disadvantages of this mode are as follows:
+
+- Advantages: 
+  1. Simple configuration.
+  2. Outage or restart(for maintenance) of one master node has no impact on the application. 
+  3. When the disk is configured as RAID10, messages are not lost because the RAID10 disk is very reliable, even if the machine is not recoverable (In the case of asynchronous flush disk mode of the message, a small number of messages are lost; If the brush mode of a message is synchronous, no message will be lost).
+  4. In this mode, the performance is the highest.
+- Disadvantages:
+  1. During a single machine outage, messages that are not consumed on this machine are not subscribed to until the machine recovers, and message real-time is affected.
+
+The starting steps for multiple Master mode are as follows:
+
+**1）Start NameServer**
+
+```shell
+### Start Name Server first
+$ nohup sh mqnamesrv &
+ 
+### Then verify that the Name Server starts successfully
+$ tail -f ~/logs/rocketmqlogs/namesrv.log
+The Name Server boot success...
+```
+
+**2）Start the Broker cluster**
+
+```shell
+### For example, starting the first Master on machine A, assuming that the configured NameServer IP is: 192.168.1.1.
+$ nohup sh mqbroker -n 192.168.1.1:9876 -c $ROCKETMQ_HOME/conf/2m-noslave/broker-a.properties &
+ 
+### Then starting the second Master on machine B, assuming that the configured NameServer IP is: 192.168.1.1.
+$ nohup sh mqbroker -n 192.168.1.1:9876 -c $ROCKETMQ_HOME/conf/2m-noslave/broker-b.properties &
+
+...
+```
+
+The boot command shown above is used in the case of a single NameServer.For clusters of multiple NameServer, the address list after the -n argument in the Broker boot command is separated by semicolons, for example, 192.168.1.1: 9876;192.161.2: 9876.
+
+### 3 Multiple Master And Multiple Slave Mode-Asynchronous replication
+
+Each Master configures a Slave, with multiple pairs of Master-Slave.HA uses asynchronous replication, with a short message delay (millisecond) between Master and Slave.The advantages and disadvantages of this mode are as follows:
```

Comment:
1. Each Master configures a Slave  => Each Master configures more than one Slave.
2. "node" is recommended to be added after "master" and "slave", this makes more precise. Of course, except proper nouns like "Multiple Master And Multiple Slave".
3. Both capitalized "Master/Slave" and uncapitalized  "master/slave" can be found. This makes a little confusing. I prefer uncapitalized ones except for titles.